docs/website/content/v0.4/en/guides/getting-started/firecracker.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Firecracker
+---
+
+In this guide we will create a Kubernetes cluster using Firecracker.
+
+## Requirements
+
+- Linux
+- a kernel with KVM enabled (`/dev/kvm` must exist)
+- at least `CAP_SYS_ADMIN` and `CAP_NET_ADMIN` capabilities
+- [firecracker](https://github.com/firecracker-microvm/firecracker/releases) (v0.21.0 or higher)
+- `bridge`, and `firewall` CNI plugins from the [standard CNI plugins](https://github.com/containernetworking/cni), and `tc-redirect-tap` CNI plugin from the [Firecracker Go SDK](https://github.com/firecracker-microvm/firecracker-go-sdk/tree/master/cni) installed to `/opt/cni/bin`
+- iptables
+- `/etc/cni/conf.d` directory should exist
+- `/var/run/netns` directory should exist
+
+## Create the Cluster
+
+```bash
+sudo talosctl cluster create --wait --provisioner firecracker
+```
+
+Once the above finishes successfully, your talosconfig(`~/.talos/config`) will be configured to point to the new cluster.
+
+## Retrieve and Configure the `kubeconfig`
+
+```bash
+talosctl kubeconfig .
+```
+
+## Using the Cluster
+
+Once the cluster is available, you can make use of `talosctl` and `kubectl` to interact with the cluster.
+For example, to view current running containers, run `talosctl containers` for a list of containers in the `system` namespace, or `talosctl containers -k` for the `k8s.io` namespace.
+To view the logs of a container, use `talosctl logs <container>` or `talosctl logs -k <container>`.
+
+A bridge interface will be created, and assigned the default IP 172.20.0.1.
```

Comment:
sorry, I confused you, default is going to be 10.5.0.1

Reply:
Fixed